linux/devdoc/completion_port_linux_requirements.md

```diff
@@ -66,19 +66,21 @@ MOCKABLE_FUNCTION(, void, completion_port_dec_ref, COMPLETION_PORT_HANDLE, compl
 
 `completion_port_dec_ref` handles the decrement of the reference count and freeing the memory if necessary.
 
-`completion_port_dec_ref` shall decrement the reference count for `completion_port`.
+**SRS_COMPLETION_PORT_LINUX_11_008: [** If `completion_port` is `NULL`, `completion_port_dec_ref` shall return. **]**
 
-If the reference count reaches 0, `completion_port_dec_ref` shall do the following:
+**SRS_COMPLETION_PORT_LINUX_11_009: [** `completion_port_dec_ref` shall decrement the reference count for `completion_port`. **]**
 
-- wait for the ongoing call count to reach zero.
+**SRS_COMPLETION_PORT_LINUX_11_010: [** If the reference count reaches 0, `completion_port_dec_ref` shall do the following: **]**
 
-- increment the flag signaling that the threads can complete.
+**SRS_COMPLETION_PORT_LINUX_11_011: [** - wait for the ongoing call count to reach zero. **]**
```

Comment:
not a big deal but my comment before was more about changing from this:

```
**SRS_COMPLETION_PORT_LINUX_11_011: [** - wait for the ongoing call count to reach zero. **]**
```

to this:

```
- **SRS_COMPLETION_PORT_LINUX_11_011: [** wait for the ongoing call count to reach zero. **]**
```

Because the markdown lists work then and the "-" doesn't need to go in the code.

Reply:
Oh, gotcha.  I will change it.